docs/sources/configure/examples/11-COS-HMAC-Example.yaml

```diff
@@ -14,7 +14,6 @@ storage_config:
   tsdb_shipper:
     active_index_directory: /loki/index
     cache_location: /loki/index_cache
-    shared_store: cos
```

Comment:
Question - If the name is COS-HMAC-Example, do we need to change the names of the example when we remove the references to COS?

Reply:
we still refer to `cos` in the `object_store`, with this both the chunks and index will be stored in the bucket pointed by `object_store`.
https://github.com/grafana/loki/pull/10840/files/a3d0395e8b827f7a9772edfe7dcf3abaca0453d9#diff-792b40fcd3a1d9d9804f34a82d2d515750d945045ea9d84d640bc2eed5056dc7L7


Similarly for other examples, we still refer to them from `object_store` in `period_config`.

Reply:
Ah, I missed that.  Should have clicked through to see the whole file.